Tests/Weaver/Realm.Fody.Tests/WeaverTests.cs

```diff
@@ -521,7 +522,16 @@ public void MatchErrorsAndWarnings()
                 "RealmSetWithSetter.People has a setter but its type is a ISet which only supports getters.",
                 "RealmDictionaryWithSetter.People has a setter but its type is a IDictionary which only supports getters.",
                 "RealmDictionaryWithNonStringKey.People is a Dictionary<Int32, Person> but only string keys are currently supported by Realm.",
-                "MixOfCollectionsObject.EmbeddedSet is a Set<EmbeddedObject> which is not supported. Embedded objects are always unique which is why List<EmbeddedObject> already has Set semantics."
+                "MixOfCollectionsObject.EmbeddedSet is a Set<EmbeddedObject> which is not supported. Embedded objects are always unique which is why List<EmbeddedObject> already has Set semantics.",
+                "Measurement.Sensor is of type AsymmetricObject, but AsymmetricObject-s aren't allowed to be the receiving end of any relationship.",
```

Comment:
Why is there a dash here?

Reply:
We usually write with the back-ticks, where it's easy to understand how the object is called -> `AsymmetricObject`s.
 I wanted to keep it simple to read when not having the back-ticks in place. Nothing else.
Should I just have AsymmetricObjects?